course_info/static/course_info/js/controllers/DetailsController.js

```diff
@@ -110,14 +110,24 @@
                 });
 
         };
-
+        dc.getCourseDescription = function(course) {
```

Comment:
Hmm, yes, it looks like `getFormattedCourseInstance()` is called from `handleCourseInstanceResponse()`, so for ILEs this may change the behavior of the form, as we discussed offline -- let's think about what we want this page to do. Maybe just let it stay blank? Seems ok to me, in this case. A comment describing why `title` differs here from the search page might be helpful, though.